assets/controller.yaml

```diff
@@ -136,5 +148,16 @@ spec:
               memory: 50Mi
               cpu: 10m
       volumes:
+        - name: aws-credentials
+          secret:
+            secretName: ebs-cloud-credentials
+        # This service account token can be used to provide identity outside the cluster.
+        # For example, this token can be used with AssumeRoleWithWebIdentity to authenticate with AWS using IAM OIDC provider and STS.
+        - name: bound-sa-token
+          projected:
+            sources:
+            - serviceAccountToken:
+                path: token
+                audience: openshift
```

Comment:
Should the credentials file in `manifests/` also be updated to use the STS credentials?

Reply:
> Should the credentials file in manifests/ also be updated to use the STS credentials?

For now no,
For 4.7, the goal is to provide a way for users to provide these STS-like credentials manually. in 4.8 when we implement the automated setup, we will be updating the CredentialRequest objects.
this PR uses a workflow for _current_ static credentials in a way that it allows manual setup today and automatic setup in 4.8

Reply:
Ack